backend/infrahub/api/schema.py

```diff
@@ -240,8 +242,17 @@ async def load_schema(
     background_tasks: BackgroundTasks,
     db: InfrahubDatabase = Depends(get_db),
     branch: Branch = Depends(get_branch_dep),
-    _: Any = Depends(get_current_user),
+    account_session: AccountSession = Depends(get_current_user),
 ) -> SchemaUpdate:
+    for permission_backend in registry.permission_backends:
+        if not await permission_backend.has_permission(
+            db=db,
+            account_id=account_session.account_id,
+            permission=f"global:{GlobalPermissions.MANAGE_SCHEMA.value}:{PermissionDecision.ALLOW.value}",
```

Comment:
not really related to this PR, but do you think it is worth adding a function that builds these permissions somewhere?
it seems like we do it a lot and it might be easier to call function than build this string every time

Reply:
Indeed, this is something we want to address at a later stage.